# Hand 목표 (ROS2 -> shm)
# Hand_Num = 2 (Left, Right)
# Hand_DOF = 16

int16[16] joint_targets        # Hand_j_tar
float64 movement_duration       # hand_movement_duration
uint8 hand_id                  # 0=Right, 1=Left

